webcomponents/tool/src/main/frontend/js/rubrics/rubrics-element.js

```diff
@@ -6,7 +6,7 @@ class RubricsElement extends SakaiElement {
 
     super();
 
-    this.locale = window.portal ? window.portal.locale : window.top.portal.locale;
+    this.locale = window.portal ? window.portal.locale : window.top.portal ? window.top.portal.locale : window.top.sakai.locale.userLocale.replace("_","-");
```

Comment:
You can simplify this with:
this.locale = (window.top?.portal?.locale || window.top?.sakai?.locale?.userLocale || "en-US").replace("_", "-");

Reply:
What that line means is:
if we have window.top.portal.locale, use it, else, if we have window.top.sakai.locale, use that, else use en-US. Then replace _ with - on whatever we end with.

Reply:
Thanks for the suggestion @adrianfish , I've just uploaded the changes.

Reply:
this is probably out of scope for this JIRA, but there is no guarantee we are window.top!!!!!